### ISSUE TITLE
Data augmentation

### ISSUE THREAD

**Issue by judithliatsf** [judithliatsf/to_distill_or_not#3]
Increase the dataset by adding pseudo labels(logits).
- Randomly loop through pair of sentences and call teacher model to get teacher_logits
- Save teacher_logits and raw pair of sentences (label is not used)
- Knowledge distillation of student model using these augmented dataset
